src/clj/ataru/forms/form_access_control.clj

```diff
@@ -44,7 +45,7 @@
    organization-service
    (fn [] (form-store/get-organization-oid-by-id form-id))))
 
-(defn post-form [form session organization-service]
+(defn- do-authenticated [form session organization-service do-fn]
```

Comment:
Better name would be "check-authorization" or similar. This doesn't authenticate in any way.